providers/flagd/src/test/java/smoketest/FFEvaluationTest.java

```diff
@@ -0,0 +1,52 @@
+package smoketest;
```

Comment:
+1 for tests

Reply:
In the long term I created [this](https://github.com/open-feature/java-sdk-contrib/issues/380).

Reply:
@thisthat I created https://github.com/open-feature/java-sdk-contrib/pull/387. This should help you and I hope shouldn't cause any conflicts for you with this PR.

Reply:
I removed the smoketest package and the dependencies on testcontainers :)